geometry_msgs/Transform transform

int32 plane_id

float64 w # width
float64 h # height 

float64 precision # fill ratio of rectangle
float64 recall # coverage ratio of rectangle in plane


